packages/contracts-bedrock/foundry.toml

```diff
@@ -7,7 +7,6 @@ optimizer_runs = 999999
 remappings = [
   '@openzeppelin/contracts-upgradeable/=node_modules/@openzeppelin/contracts-upgradeable/',
   '@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/',
-  '@eth-optimism/contracts-periphery/=node_modules/@eth-optimism/contracts-periphery/contracts',
```

Comment:
This was wrong (should remap to `../../@eth-optimism`), but it doesn't appear to be in use so I've just removed it.

Reply:
you cannot import remappings outside of the current dir, the base layer `package.json` hoists `contracts-periphery` into the local node modules here, if `contracts-periphery` is no longer a dep to `contracts-bedrock` then we can delete the hoisting in the base `package.json`

Reply:
In the end it seems that: 
1. remappings outside the current dir (as above) do work
2. despite [this change](https://github.com/ethereum-optimism/optimism/pull/4617/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R28) (which I thought would properly apply `nohoist` to the `contracts-bedrock` package), those contracts are still found in the root `node_modules` dir. I have checked that this is the case both locally and by ssh'ing into CI. 

It all seems a bit odd, but given the checks are now passing I'm going to resolve which will hopefully unblock the release process.